pkgs/applications/graphics/vimiv-qt/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python39Packages
+, qt5
```

```suggestion
, qt5
, installShellFiles
```

required for installManPage

---

Done with the last commit, sorry forgot to resolve the conversation.